recipe/meta.yaml

```diff
@@ -15,16 +15,16 @@ source:
 build:
   noarch: python
   number: 0
-  script: python setup.py install --single-version-externally-managed --record=record.txt
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
 
 requirements:
   build:
     - python
-    - setuptools
+    - pip
 
   run:
     - python
-    - spyder >=3.2
+    - spyder >=4, <5
```

Comment:
I think you don't need to leave a space after the comma here. So this should be

    - spyder >=4,<5